entity-framework/core/what-is-new/ef-core-10.0/breaking-changes.md

```diff
@@ -83,3 +86,95 @@ await context.Blogs.ExecuteUpdateAsync(s =>
     }
 });
 ```
+
+<a name="MDS-breaking-changes"></a>
+
+## Microsoft.Data.Sqlite breaking changes
+
+### Summary
+
+| **Breaking change**                                                                                       | **Impact** |
+|:----------------------------------------------------------------------------------------------------------|------------|
+| [Reading DateTimeOffset without an offset now assumes UTC](#DateTimeOffset-read)                          | High       |
+| [Writing DateTimeOffset into REAL column now writes in UTC](#DateTimeOffset-write)                        | High       |
+| [Reading DateTime with an offset now returns value in UTC](#DateTime-read)                                | High       |
+
+### High-impact changes
+
+<a name="DateTimeOffset-read"></a>
+
+#### Reading DateTimeOffset without an offset now assumes UTC
+
+[Tracking Issue #36195](https://github.com/dotnet/efcore/issues/36195)
+
+##### Old behavior
+
+Previously, when reading a `DateTimeOffset` value that did not have an offset (e.g., `2014-04-15 10:47:16`), Microsoft.Data.Sqlite would assume the value was in the local time zone.
```

Comment:
> a `DateTimeOffset` value that did not have an offset

That seems like a bit of a contradiciton in terms... SQLite doesn't actually store a `DateTimeOffset` - if stores the textual representation of a timestamp.

Also, it's not quite clear to me what this "assumption" entails - consider documenting the user-visible effect, i.e. how the values read are actually different. Best to also give an example of the same value in the database, and what you get when you read it before and after.